rocoto/cronjob_hafs_datm.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
```

Comment:
@evankalina or @SamuelTrahanNOAA, could you please merge cronjob_hafs_datm.sh and cronjob_hafs_docn.sh into one file, saying cronjob_hafs_cdeps.sh? Thanks!